### DISCUSSION TITLE
How can I get trayIcon run xfreerdp on Ubuntu ?

### DISCUSSION THREAD

**Discussion by JackZ0** [FreeRDP/FreeRDP#7371] [Q&A]
How can I get trayIcon run xfreerdp on Ubuntu ? I hope I don't hesitate to enlighten me

**Comment by akallabeth** [comment]
What do you mean?
1. A tray icon when running a desktop session -> that is not supported
2. A tray icon in a RAILS / remote app session? That should already be shown if it is started.

**Reply by JackZ0** [reply]
I mean 2.

**Reply by akallabeth** [reply]
Just to be sure, the tray icon is a windows application tray icon, not one for `xfreerdp` itself.

**Reply by JackZ0** [reply]
> Just to be sure, the tray icon is a windows application tray icon, not one for `xfreerdp` itself.

thank you for your reply! But the trayIcon did not show it, for example, WeChat ,Tencent qq   or other software cannot get message option. I hope I don't hesitate to enlighten me!

**Reply by JackZ0** [reply]
> Just to be sure, the tray icon is a windows application tray icon, not one for `xfreerdp` itself.

I just want to display windows application tray icon on ubuntu.  just like ubuntu run windows application ！ I don’t know if my expression is cleared.

**Reply by akallabeth** [reply]
@JackZ0 just tried it, there seems to be some issue (configuration or bug in freerdp). Will get back when I have time to test this out.

**Reply by JackZ0** [reply]
> @JackZ0 just tried it, there seems to be some issue (configuration or bug in freerdp). Will get back when I have time to test this out.

Thanks !
